src/main/scala/com/microsoft/ml/spark/codegen/Wrappable.scala

```diff
@@ -0,0 +1,392 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark.codegen
+
+import java.lang.reflect.{ParameterizedType, Type}
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+
+import com.microsoft.ml.spark.core.env.FileUtilities
+import com.microsoft.ml.spark.core.serialize.ComplexParam
+import jdk.jshell.spi.ExecutionControl.NotImplementedException
+import org.apache.spark.ml.evaluation.Evaluator
+import org.apache.spark.ml.{Estimator, Transformer}
+import org.apache.spark.ml.param._
+
+import scala.collection.Iterator.iterate
+import scala.reflect.ClassTag
+
+object CodegenUtils {
+  def indent(test: String, numTabs: Int): String = {
```

Comment:
change test:  to content: (or value or so)

Reply:
done